src/cli/commands/bitswap.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = {
+  command: 'bitswap',
+
+  description: 'A set of commands to manipulate the bitswap agent.',
+
+  builder (yargs) {
+    return yargs
+      .commandDir('bitswap')
```

Comment:
Is building the subcommands inside the main command a best practice? Didn't see it on the module description.